packages/tabs/tabs.scss

```
@@ -0,0 +1,52 @@
+@use "~@fremtind/jkl-core/jkl";
+
+$tab-text-color: jkl.$color-granitt;
+$tab-text-color--inverted: jkl.$color-snohvit;
+
+$tab-indicator-color: jkl.$color-granitt;
+$tab-indicator-color--inverted: jkl.$color-snohvit;
+
+$tab-hover-color: jkl.$color-stein;
+$tab-hover-color--inverted: jkl.$color-svaberg;
+
+@include jkl.helper-light-mode-variables {
+    --tab-text-color: #{$tab-text-color};
+    --tab-indicator-color: #{$tab-indicator-color};
+    --tab-hover-color: #{$tab-hover-color};
+}
+
+@include jkl.helper-dark-mode-variables {
+    --tab-text-color: #{$tab-text-color--inverted};
+    --tab-indicator-color: #{$tab-indicator-color--inverted};
+    --tab-hover-color: #{$tab-hover-color--inverted};
+}
+
+.jkl-tablist {
+    position: relative;
+    display: flex;
+    flex-direction: row;
+    padding: 0;
+    margin: 0;
+    border-bottom: 1px jkl.$color-svaberg solid;
+    width: 100%;
+}
+
+.jkl-tab {
+    @include jkl.text-style("body");
+    border: none;
+    background-color: transparent;
+    padding: jkl.$spacing-m;
+    cursor: pointer;
+    color: var(--tab-text-color);
```

Husker ikke om vi har vedtatt å droppe støtte for IE i Jøkul, men hvis ikke trengs det statiske verdier som backup der det er brukt custom properties:
```suggestion
    color: $tab-text-color;
    color: var(--tab-text-color);
```

---

Vi dropper det for nye ting i hvertfall, og ting som er merket som deprecated (`--inverted`) fjernes typisk ved en anledning om vi uansett får en breaking change.

---

Har vi sagt klart fra om det? Spør siden jeg tydeligvis ikke har fått det med meg 😅 Hvis det er løsninger hos oss som fortsatt støtter IE er det i det minste greit at de vet hva som evt. vil knekke løsningen, så kanskje man burde hatt en slags tag for det på komponentene det gjelder?

---

Mener det var noe vi ble enige om på forum, i #2374  og med https://jokul.fremtind.no/blog/bye-ie/ ?

---

Vet ikke om det har blitt vedtatt allerede, men vi kan gjerne vedta det snart! (eventuelt nå)

---

Jeg tror vi må gjøre dette klarere i så fall. #2374 er jo fortsatt kompatibel med IE, den bare viser ikke gjenstående tegn. Denne komponenten vil derimot mangle grunnleggende utseende i IE.

Jeg tolket bloggposten som en guide til hvordan teamene kan slutte å støtte IE, men jeg synes at så lenge ikke alle team _har_ sluttet bør vi enten fortsette å støtte det i Jøkul (evt. med progressive enhancement som i TextArea) eller gi klar beskjed om hvilke komponenter som faktisk ikke kan brukes for IE.

Helt enig i at vi bør få en offisiell slutt på støtte snart, men det bør nok gjøres litt mer offentlig og organisert IMO 😊

---

Helt enig i at det må gjøres offentlig og organisert. 
Jeg kan høre på stand-up i dag hvor mange team som har laget sperren og hvem som manger, så kanskje vi får noen sva